instance/models/mixins/openedx_theme.py

```diff
@@ -115,14 +106,52 @@ def get_theme_settings(self):
             ],
             "SIMPLETHEME_ENABLE_DEPLOY": True,
             "EDXAPP_DEFAULT_SITE_THEME": "simple-theme",
+            "EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO": settings.SIMPLE_THEME_SKELETON_THEME_REPO,
+            "EDXAPP_COMPREHENSIVE_THEME_VERSION": settings.SIMPLE_THEME_SKELETON_THEME_VERSION,
             "SIMPLETHEME_EXTRA_SASS": """
-                .global-header {{
-                    background: {header_bg};
-                }}
-                .wrapper-footer {{
-                    background: {footer_bg};
-                }}""".format(header_bg=application.header_bg_color,
-                             footer_bg=application.footer_bg_color)
+                $main-color: {main_color};
```

Comment:
@giovannicimolin, since this is only defining new variables, can't these be set in the `SIMPLETHEME_SASS_OVERRIDES` above?

Reply:
@lgp171188 I can't do that, because I'm not able to access those values on the overrides when I tried at first.
I'll give this a couple more tries, but if I don't succeed I'll leave this as is :grin:

Reply:
@lgp171188 I had to leave them as is. :disappointed: